test/day04.test.js

```diff
@@ -0,0 +1,52 @@
+const { expect } = require('chai');
+const { passwordFinder1 } = require('../day04/passwordFinder1');
+const { passwordFinder2 } = require('../day04/passwordFinder2');
```

Comment:
This is so awesome you wrote tests!!!! Maybe when you do your preparatory refactoring, you can do some Test driven development as you go! Although since most of your code is written maybe that's better suited to an exercise next time!